### PR TITLE
Closes #546: Write concept importer workflow integration tests

### LINKED ISSUE #546: Write workflow integration tests for import concept workflow
There is no test for workflow:
/iis-wf-import/src/main/resources/eu/dnetlib/iis/wf/importer/concept/oozie_app/workflow.xml

### PR DESCRIPTION
**Pull request by marekhorst**

### CODE REVIEW

File: iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/StaticResourcesProvider.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    /**
+     * List of resources to be provided.
+     */
+    private final List<String> resources;
+    
+    //------------------------ CONSTRUCTORS --------------------------
+    
+
+    /**
+     * @param resourcesLocations resources to be read classpath locations
```
Comment (lukdumi) [inline iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/StaticResourcesProvider.java]: a strange comment, maybe: classpaths of resources to read

File: iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/StaticResourcesProvider.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+     */
+    public StaticResourcesProvider(String ...resourcesLocations) {
```
Comment (lukdumi) [inline iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/StaticResourcesProvider.java]: resourceClasspaths would tell at one what it is

Comment (lukdumi) [inline iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/StaticResourcesProvider.java]: And the main thing here: I don't think you need to do these io operations in constructor. In my view, you should have written a static method that reads the properties from the given classpaths, that's all. It's just an ordinary util class, isn't it?

File: iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/concept/ConceptXmlHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        assertEquals(3, receiver.getConcepts().get(2).getParams().size());
+        assertEquals("7.A.SP1.03.19.01", receiver.getConcepts().get(2).getParams().get("CD_DIVNAME"));
+        assertEquals("ICT-2011.9.1", receiver.getConcepts().get(2).getParams().get("CD_ABBR"));
+        assertEquals("7.A.SP1.03.19", receiver.getConcepts().get(2).getParams().get("CD_PARENT_DIVNAME"));
+        
+    }
+    
+ //------------------------ INNER CLASS --------------------------
+    
+    private static class ConceptsReceiver implements RecordReceiver<Concept> {
```
Comment (lukdumi) [inline iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/concept/ConceptXmlHandlerTest.java]: Creating the ConceptReceiver is unnecessary and redundant. You could mock the concept receiver and then capture the argument passed to the 'receive' method (see the Mockito ArgumentCaptor).

Comment (marekhorst) [inline iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/concept/ConceptXmlHandlerTest.java]: True, forgot about this nice feature.

**Pull request closed without merging**